src/routers/Stripe/common.ts

```diff
@@ -0,0 +1,130 @@
+import assert from 'assert';
+import { ContextType } from '../../context/Context';
+import { Order } from 'src/modules/order/Models/Order';
+import { WhpptMongoDatabase } from 'src/Services/Database/Mongo/Database';
+
+export type CalculateTotalArgs = (
```

Comment:
Can this file be split into seperate files and not in common?

Reply:
done